src/frontend/apps/e2e/__tests__/app-desk/mail-domain-create-mailbox.spec.ts

```diff
@@ -269,4 +289,254 @@ test.describe('Mail domain create mailbox', () => {
+        await route.fulfill({
+          json: {
+            count: mailDomainsFixtures.length,
+            next: null,
+            previous: null,
+            results: mailDomainsFixtures,
+          },
+        });
+      });
+      await page.route('**/api/v1.0/mail-domains/domainfr', async (route) => {
+        await route.fulfill({
+          json: mailDomainDomainFrFixture,
+        });
+      });
+      await page.route(
+        '**/api/v1.0/mail-domains/domainfr/mailboxes/?page=1**',
+        async (route) => {
+          await route.fulfill({
+            json: {
+              count: mailboxesFixtures.domainFr.page1.length,
+              next: null,
+              previous: null,
+              results: mailboxesFixtures.domainFr.page1,
+            },
+          });
+        },
+        { times: 1 },
+      );
+
+      await page.route(
+        '**/mail-domains/domainfr/mailboxes/',
+        async (route) => {
+          if (route.request().method() === 'POST') {
+            if (countCreateMailboxRequestSent === 0) {
+              await route.fulfill({
+                status: 400,
+                json: {
+                  local_part: ['Enter a valid value.'],
+                  secondary_email: ['Enter a valid email address.'],
+                },
+              });
+            }
+
+            if (countCreateMailboxRequestSent === 1) {
+              await route.fulfill({
+                status: 400,
+                json: {
+                  local_part: [
+                    'Mailbox with this Local_part and Domain already exists.',
+                  ],
+                },
+              });
+            }
+
+            countCreateMailboxRequestSent++;
+          }
+        },
+        { times: 2 },
+      );
+    };
+
+    await interceptApiCalls();
+
+    await clickOnMailDomainsNavButton(page);
+
+    await page.getByRole('listbox').first().getByText('domain.fr').click();
+
+    await page.getByRole('button', { name: 'Create a mailbox' }).click();
+    await page.getByRole('button', { name: 'Cancel' }).click();
+
+    await expect(page.getByTitle('Mailbox creation form')).toBeHidden();
+
+    await page.getByRole('button', { name: 'Create a mailbox' }).click();
+
+    await expect(page.getByTitle('Mailbox creation form')).toBeVisible();
+    await expect(
+      page.getByRole('heading', { name: 'Create a mailbox' }),
+    ).toBeVisible();
+
+    const inputFirstName = page.getByLabel('First name');
+    const inputLastName = page.getByLabel('Last name');
+    const inputMainEmailAddress = page.getByLabel('Main email address');
+    const inputSecondaryEmailAddress = page.getByLabel(
+      'Secondary email address',
+    );
+    const submitButton = page.getByRole('button', { name: 'Submit' });
+
+    await inputFirstName.fill('John');
+    await inputLastName.fill('Doe');
+    await inputMainEmailAddress.fill('wrong@');
+    await expect(page.getByText('do not include "@" in it')).toBeVisible();
```

Comment:
Try to check only what this test is about, I have the feeling that it is the same assertion has the previous test.